### DISCUSSION TITLE
Trouble with keyframes

### DISCUSSION THREAD

**Discussion by Ballistictoon** [keiwando/toonsquid#661] [Q&A]
I’m having trouble using keyframes. They aren’t moving when I click them and when I move a frame, it just moves all frames in that layer, anyone know what’s going on?

**Comment by DreamBagAttic** [comment]
Did you click the button to add a keyframes where you're changing things? You probably did do this but I was having this problem when I was learning and honestly didn't realize I had to manually add keyframes first

**Reply by Ballistictoon** [reply]
I know of the keyframes part where it says “add keyframe” But when I move a layer or image, unless I press duplicate and move every single frame along the line, it doesn’t move when I press play

**Reply by Ballistictoon** [reply]
Sorry if I’m new to this and just being stupid with something obvious btw

**Comment by budaloops** [comment]
Can you share a screenshot or a video?

**Comment by keiwando** [comment]
This might help:
https://toonsquid.com/handbook/guides/faq/#pixel-layer-keyframes
